File: POH6/tsubame/Main.java
/*
 * 結果 https://paiza.jp/poh/joshibato/tsubame/result/ee79f099
 */
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner in = new Scanner(System.in);
        
        String n_str = in.next();
        
        int n = Integer.parseInt(n_str);
        int n1 = Integer.parseInt(n_str.substring(1,2));
        int n10 = Integer.parseInt(n_str.substring(0,1));
        int r = n + n1 + n10;
        
        System.out.println(r);
        
    }
}

File: POH6/tsubame/Main.scala
/*
 * 結果 https://paiza.jp/poh/joshibato/tsubame/result/4d3f5f16
 */
object Main extends App{
    
    val n_str = scala.io.StdIn.readLine()
    
    val n = n_str.toInt
    val n1 = n_str.substring(1,2).toInt
    val n10 = n_str.substring(0,1).toInt
    val r = n + n1 + n10
    
    println(r)
}
